showcase/tests/integration/components/hds/pagination/compact-test.js

```diff
@@ -89,12 +95,21 @@ module('Integration | Component | hds/pagination/compact', function (hooks) {
   });
 
   test('it renders the passed in currentPageSize value', async function (assert) {
+    this.set('currentPageSize', 40);
+    this.set('myQueryFunction', (page, pageSize) => ({ page, pageSize }));
+
     await render(hbs`
-      <Hds::Pagination::Compact @showSizeSelector={{true}} @currentPageSize={{40}} @pageSizes={{array 20 40 60}} />
+      <Hds::Pagination::Compact @showSizeSelector={{true}} @currentPageSize={{this.currentPageSize}} @pageSizes={{array 20 40 60}}  @route="components.pagination" @queryFunction={{this.myQueryFunction}} />
```

Comment:
Question: for both pagination component tests, in order to test that the currentPage updates appropriately when you change the value, I added `@route` and `@queryFunction` because those are required to make the component controlled. I believe this is ok because if the consumer is controlling the currentPage attribute from the outside, the whole thing should be controlled, right?

Reply:
I think you're assumption is correct, but I'd like to get @didoo's opinion on this as well, as he's more familiar with this component than I am

Reply:
That's also my understanding